app/src/main/res/values/strings.xml

```diff
@@ -226,6 +226,11 @@
   <string name="profile_edit_delete_dialog_negative">Cancel</string>
   <string name="profile_edit_allow_download_heading">Allow Download Access</string>
   <string name="profile_edit_allow_download_sub">User is able to download and delete content without Administrator password</string>
+  <!-- ProfileProgressActivity. -->
```

Comment:
Remobve `.` at the end I know there are lines in this file which have this full-stop but they are incorrect.
`<!-- ProfileProgressActivity -->`